src/modules/LanItems.rb

```diff
@@ -1529,24 +1445,24 @@ def SelectHWMap(hardware)
       # Wireless Card Features
       @wl_auth_modes = Builtins.prepend(
         Convert.convert(
-          Ops.get(hardware, "wl_auth_modes", @nilliststring),
+          Ops.get(hardware, "wl_auth_modes", nil),
```

Comment:
yep, this was fix by s/../../g targeted against the `@nilstring` variable, so I didn't check all replacements. 

Wireless configuration is wild field which will need lot of love. I works, at least somehow, now (most probably thanks to NetworkManager :-/)

I'll tweak it a bit more regarding dropping some more builtins.